src/backend/dx11/src/dxgi.rs

```diff
@@ -105,15 +105,16 @@ fn create_dxgi_factory1(guid: &GUID) -> Result<ComPtr<dxgi::IDXGIFactory>, winer
 pub(crate) fn get_dxgi_factory() -> Result<(ComPtr<dxgi::IDXGIFactory>, DxgiVersion), winerror::HRESULT> {
     let mut factory: *mut IUnknown = ptr::null_mut();
 
-    if let Ok(factory) = create_dxgi_factory2(&dxgi1_5::IDXGIFactory5::uuidof()) {
+    // TODO: do we even need `create_dxgi_factory2`?
```

Comment:
let's double check the PR that started using factory2

Reply:
The only thing different with `CreateDXGIFactory2` is that you can specify `DXGI_CREATE_FACTORY_DEBUG`, which gets set by the runtime anyway: https://msdn.microsoft.com/en-us/library/windows/desktop/dn268307(v=vs.85).aspx

Using `CreateDXGIFactory1` will still return the interfaces from later versions.